components/automate-gateway/handler/authz.go

```diff
@@ -18,19 +18,18 @@ import (
 
 	// Gateway Requests/Response/Service definitions
 	"github.com/chef/automate/components/automate-gateway/api/authz/pairs"
-	"github.com/chef/automate/components/automate-gateway/api/authz/policy"
 	gwAuthzReq "github.com/chef/automate/components/automate-gateway/api/authz/request"
 	gwAuthzRes "github.com/chef/automate/components/automate-gateway/api/authz/response"
 
 	// other dependencies
-	"github.com/chef/automate/components/automate-gateway/authz/policy_v2"
+	policy "github.com/chef/automate/components/automate-gateway/authz/policy_v2"
 	"github.com/chef/automate/components/automate-gateway/gateway/middleware"
 	"github.com/chef/automate/lib/grpc/auth_context"
 )
 
 // AuthzServer is the server interface
 type AuthzServer struct {
-	client        authz.AuthorizationClient
+	client        authz.AuthorizationClient // now only needed for GetVersion
```

Comment:
Another bit of future work. We still need a v1 client here for GetVersion and for introspection calls; but the v1 client also handles policies here, too(!), hence my comments below in this file.

Reply:
Can we make a card for this?

Reply:
Done: https://github.com/chef/automate/issues/2808